inc/define.php

```diff
@@ -591,7 +591,10 @@
     ],
     'admin'        => ['clipboard', 'sortable'],
     'preference'   => ['clipboard'],
-    'self-service' => array_merge(['tinymce'], $reservations_libs)
+    'self-service' => array_merge(['tinymce'], $reservations_libs),
+    'tickets'      => [
```

Comment:
`front/ticket.php` is the only self-service page that use its own menu, allowing it to have a dedicated breadcrumb:

![image](https://user-images.githubusercontent.com/42734840/173577300-0a81e6d4-3360-4568-981f-e2028e55e211.png)

IMO this page isn't the problem, in fact it's the only correct one.
Every other self-service pages should have a dedicated breadcrumb too instead of showing only "home":

![image](https://user-images.githubusercontent.com/42734840/173577600-0a777b73-0dc9-4f3e-b40e-d329f0ffb9be.png)
(This is the current reservation page)